src/pages/steps/AdditionalInformationStep/BookingInfoStep.tsx

```diff
@@ -132,14 +132,11 @@ const ReservationPeriod = ({
     );
   };
 
-  const handleNewBookingPeriod = async (
-    startDate: Date,
-    endDate: Date,
-  ): Promise<void> => {
-    await handlePeriodChange(endDate, startDate);
+  const handleNewBookingPeriod = (startDate: Date, endDate: Date): void => {
+    handlePeriodChange(endDate, startDate);
   };
 
-  const handleChangeStartDate = async (newStartDate: Date): Promise<void> => {
+  const handleChangeStartDate = (newStartDate: Date): Promise<void> => {
```

Comment:
We can maybe remove the manually hinted return types in those three functions, since TS can likely infer them and I don't think it returns a Promise anyway no?

Reply:
Done